Quaver.Shared/Graphics/Graphs/DifficultySeekBar.cs

```diff
@@ -120,7 +120,8 @@ public override void Update(GameTime gameTime)
                 var percentage = (MouseManager.CurrentState.Y - AbsolutePosition.Y) / AbsoluteSize.Y;
                 var targetPos = (1 - percentage) * Track.Length;
 
-                SeekToPos(targetPos);
+                if (Track.Length - targetPos > 1000)
```

Comment:
Can this now also be removed? It kinda prevents you skipping to the end of the track in seekbar. Which is minor thing but I tested removing it, it does not change anything.

Reply:
Dropped the corresponding commit as this bug doesn't show now on my side either.